walsh_a07b.js

```diff
@@ -0,0 +1,129 @@
+/* Tyler Walsh
+walsh_ao7b.js
+19WI_INFO_2124_WW
+Thoendel
+1/30/2019 */
+
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+const standardInput = process.stdin;            
+standardInput.resume();
+standardInput.setEncoding('utf8');
+const badFoods = [];
+const groceryItems = [];
+clearScreen();
+console.log(getPrompt(groceryItems));
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+
+//Step 1
+buildFoodsList(badFoods, "bread", "beer", "bacon", "butter", "beans"); //Invoking the buildFoodsList function to add these 5 items to the bad foods array 
+
+standardInput.on('data', function (text) {
+    //This line will remove line breaks 
+    //\n\r in window, \r in macOS
+    text = text.replace('\n', '').replace('\r', '');
+    //Step 4 Begin
+    if(text == "q" || text == "Q") { //If the user inputs a q in order to close the program
+        clearScreen(); //Clear the screen
+        console.log("Bye."); //Tell the user that their action was registered.
+        process.exit(); //Exit the process.
+
+    } else if (text == "v" || text == "V") {
+
+        clearScreen(); //clear the screen
+
+        displayList(groceryItems); //Show the user what is in their grocery list.
+
+        console.log(getPrompt(groceryItems)); //Then prompt the user to continue.
+
+    } else { //If the user did not try to exit the program.
+
+        clearScreen(); //clear the screen
+
+        if (checkItem(text, badFoods) == false) { //Compare the user's input to the list of bad foods. If the input is not a bad food do this.
+        //Note: Putting spaces before a "bad food" still causes it to be added to the list. This could be fixed by moving the formatItem function to before the if statement, however I chose to follow the instructions closely.

```

Comment:
solid point! I would have supported that improvement - but mostly I'm glad that you see that option! 🥇